### PR TITLE
#167891561 Setup PostgreSQL Database 

### PR DESCRIPTION
**Pull request by femitj**
#### What does this PR do?
This PR setup PostgreSQL database

#### Description of Task to be completed?
 * Setup express server
 * Setup babel 
 * Setup Postgres

#### How should this be manually tested?
Clone this repo https://github.com/andela/banshee-bn-backend into any folder of your choice.
Cd into the clone folder
Install dependencies with npm install
Run npm run start:dev

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167891561](https://www.pivotaltracker.com/story/show/167891561)

#### Screenshots
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by Pomile — COMMENTED** [review]
I noticed you used session in the app(index.js - server entry point). I don't think we need to use session in this regard since we are using json web token.

**Comment by femitj** [comment]
Yes you're right @Pomile, i did not include that myself as it was already in the repo
If i'm given authority in our next meeting, i would like to clean up alot of things already included.

**Comment by thatkazuk1** [comment]
1. Where you asked for *relevant pivotal stories*, I believe the right thing is to insert the PT ID instead of providing a link as you did.
2. `body-parser` middleware is also unnecessary at this point:
``` js
app.use(express.json());
app.use(express.urlencoded({ extended: false });
```
will suffice here

####
_I also made some changes to  `index.js` so I don't know how we can resolve the conflict that will arise_

**Comment by thatkazuk1** [comment]
> Yes you're right @Pomile, i did not include that myself as it was already in the repo

Concerning this, you can start with a barebone template and then add things as necessary in the future

**Comment by Pomile** [comment]
Ok. Cool

**Comment by femitj** [comment]
okay thanks @meetKazuki, I've taken note of that

**Comment by thatkazuk1** [comment]
> okay thanks @meetKazuki, I've taken note of that

Also include the PT ID in the title of PR. Something like this `<PT ID> Setup PostgreSQL Database`

**Review by Pomile — APPROVED** [review]

**Review by thatkazuk1 — CHANGES_REQUESTED** [review]
Have you implemented said changes @femitj?

**Comment by femitj** [comment]
> Have you implemented said changes @femitj?

pls refer to the changes you requested?

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,61 @@
...
 // Normal express config defaults
-app.use(require("morgan")("dev"));
+//app.use(require("morgan")("dev"));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
+app.use(require('method-override')());
+app.use(express.static(__dirname + '/public'));
```
Comment (tvpeter) [inline src/index.js:20]: Do we need static files directory? I don't think so since it's an api

File: src/index.js [hunk 2]
```diff
@@ -1,93 +1,61 @@
...
+});
 
 app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
+  session({
```
Comment (tvpeter) [inline src/index.js:27]: We might not need sessions.

**Review by tvpeter — COMMENTED** [review]
Kindly remove the sessions and static files directory. Thank you

**Pull request closed without merging**